chapters/annotations.tex

```diff
@@ -383,20 +383,25 @@ \section{Annotations for Simulations}\label{annotations-for-simulations}
 \end{center}
 
 \begin{annotationdefinition}[experiment]
-% henrikt-ma 2021-04: Seems strange to allow 'experiment' completely without list of options -- what would it mean?
 \begin{synopsis}[grammar]\begin{lstlisting}
-"experiment"
-   [ "(" [ experimentOption { "," experimentOption } ] ")" ]
-
-experimentOption:
-   "StartTime" "=" [ "+" | "-" ] UNSIGNED-NUMBER
-   | "StopTime" "=" [ "+" | "-" ] UNSIGNED-NUMBER
-   | "Interval" "=" UNSIGNED-NUMBER
-   | "Tolerance" "=" UNSIGNED-NUMBER
+annotation "experiment" "(" experimentModifiers ")"
 \end{lstlisting}\end{synopsis}
 \begin{semantics}
-The \lstinline{experiment} annotation defines the default start time (\lstinline!StartTime!) in {[}s{]}, the default stop time (\lstinline!StopTime!) in {[}s{]}, the suitable time resolution for the result grid (\lstinline!Interval!) in {[}s{]}, and the default relative integration tolerance (\lstinline!Tolerance!) for simulation experiments to be carried out with the model or block at hand.
+The modifiers are for the following record.
+\begin{lstlisting}[language=modelica]
+record experiment
+  Real StartTime(unit = "s") = 0;
+  Real StopTime(unit = "s");
+  Real Interval(unit = "s") = $\ldots{}$;
+  Real Tolerance = $\ldots{}$;
+end experiment;
+\end{lstlisting}
+The \lstinline!experiment! annotation defines the start time (\lstinline!StartTime!) in {[}s{]}, the stop time (\lstinline!StopTime!) in {[}s{]}, the suitable time resolution for the result grid (\lstinline!Interval!) in {[}s{]}, and the relative integration tolerance (\lstinline!Tolerance!) for simulation experiments to be carried out with the model or block at hand.
```

Comment:
Two reasons to not say _the model or block at hand_:
- We should generally make use of the term _simulation model_ instead.
- In this particular case we should leave the door open for inheritance, meaning that the `experiment` may reside in a non-simulation model.
```suggestion
The \lstinline!experiment! annotation defines the start time (\lstinline!StartTime!) in {[}s{]}, the stop time (\lstinline!StopTime!) in {[}s{]}, the suitable time resolution for the result grid (\lstinline!Interval!) in {[}s{]}, and the relative integration tolerance (\lstinline!Tolerance!) for simulation experiments.
```

Reply:
I hope this does not count the same as being specified on the model containing the single component, but I can imagine that looking at the `experiment` annotations of the components in a model could be a useful way to heuristically obtain settings that a tool can suggest that the user set in case no `experiment` annotation is found. Anyway, if we'd like to discuss this further, I think it belongs more to #2314 than the present PR.

Reply:
I'm of the opinion that as long as #2535 is being discussed, we shouldn't make clear statements that either speak in favor or against inheritance.

Reply:
But *the marked text wasn't changed in this PR*.
I can understand changing it - but that should be in a separate PR - like #2535; not in this one.

Blocking PRs by requesting unrelated changes isn't helpful.